.azure-pipelines/1esmain.yml

```diff
@@ -47,6 +47,8 @@ extends:
     sdl:
       credscan:
         suppressionsFile: $(Build.SourcesDirectory)\.azure-pipelines\compliance\CredScanSuppressions.json
+      componentgovernance:
+        ignoreDirectories: $(Build.SourcesDirectory)\**\.vscode-test
```

Comment:
I don't know if this `**\` syntax will work. If not, we will have to explicitly exclude all of the packages' `.vscode-test` directories.